README.md

```diff
@@ -4,7 +4,8 @@
 [![mobile_scanner](https://github.com/juliansteenbakker/nordic_dfu/actions/workflows/flutter_format.yml/badge.svg)](https://github.com/juliansteenbakker/nordic_dfu/actions/workflows/flutter_format.yml)
 [![GitHub Sponsors](https://img.shields.io/github/sponsors/juliansteenbakker?label=sponsor%20me%20)](https://github.com/sponsors/juliansteenbakker)
 
-## 5.0.0 Breaking changes!
+## 5.3.0 Breaking changes!
+From version 5.3.0, macos implimentation.
```

Comment:
It's not a breaking change, so we can just mention that there is support from version 6.0.0.

Reply:
Made the requested changes to the readme file.